detox/e2e/test/channels/create_public_channel.e2e.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {toChannelScreen} from '@support/ui/screen';
+
+import {Setup} from '@support/server_api';
+
+describe('Channels', () => {
+    beforeAll(async () => {
+        const {user} = await Setup.apiInit();
+
+        await toChannelScreen(user);
+    });
+
+    it('MM-T3201 Create public channel', async () => {
+        // # Go to channel sidebar list
+        await element(by.id('channel_drawer_button')).tap();
+
+        // # Tap on + public channels
+        await element(by.id('action_button_sidebar.channels')).tap();
+
+        // * Expect a list of public channels, initially empty
+        await expect(element(by.text('No more channels to join'))).toBeVisible();
+
+        // # Tap to create new channel
+        await element(by.id('public_channels.create.button')).tap();
+
+        // * Expect a new screen to create a new public channel
+        await expect(element(by.text('New Public Channel'))).toBeVisible();
+
+        // # Fill data
+        await element(by.id('edit_channel.name.input')).typeText('a');
+        await element(by.id('edit_channel.create.button')).tap();
```

Comment:
This failed on iOS. Please change to:
```javascript
import jestExpect from 'expect';
...
        if (device.getPlatform() === 'ios') {
            const attributes = await element(by.id('edit_channel.create.button')).getAttributes();
            jestExpect(attributes.visible).toEqual(true);
            jestExpect(attributes.enabled).toEqual(false);
        } else {
            await element(by.id('edit_channel.create.button')).tap();
        }
```

Reply:
done! it works in android, please could you tell me if it works in ios too?